pkg/controllers/nodeclass/status/ami_test.go

```diff
@@ -534,4 +534,111 @@ var _ = Describe("NodeClass AMI Status Controller", func() {
 		nodeClass = ExpectExists(ctx, env.Client, nodeClass)
 		Expect(nodeClass.StatusConditions().IsTrue(v1.ConditionTypeAMIsReady)).To(BeFalse())
 	})
+	Context("NodeClass AMI Status", func() {
```

Comment:
Can we also add a test in our integration suite? https://github.com/aws/karpenter-provider-aws/blob/main/test/suites/ami/suite_test.go

Reply:
Added the status conditions checks to existing tests